les/api_backend.go

```diff
@@ -209,6 +210,11 @@ func (b *LesApiBackend) ServiceFilter(ctx context.Context, session *bloombits.Ma
 	}
 }
 
-func (b *LesApiBackend) GasFeeRecipient() common.Address {
+func (b *LesApiBackend) GatewayFeeRecipient() common.Address {
 	return b.eth.GetRandomPeerEtherbase()
 }
+
+func (b *LesApiBackend) GatewayFee() *big.Int {
+	// TODO(nategraf): Create a method to discover the gateway fee values of peers.
```

Comment:
Definitely out of scope of this PR, but we can do this in the same message that we send to get the etherbase of our peers

Reply:
Adjusted to the TODO message to include that.